.vale/fixtures/RedHat/TermsErrors/testvalid.adoc

```diff
@@ -358,6 +358,7 @@ kernel oops
 kernel-space
 keyboard shortcut
 knowledge base
+kube-burner
```

Comment:
It's the name of a [tool](https://kube-burner.github.io/kube-burner/latest/), so I think it should be capitalized.

Reply:
The terms errors rule as-is will catch "Kube" and "kube".

`ignorecase: true`